### PR TITLE
Notifications to PF4 and some related fixes

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
Before: 
Screen Shot 2019-11-12 at 22 15 39
After:
Screen Shot 2019-11-12 at 22 53 43

Some more examples:
-----
Screen Shot 2019-11-12 at 22 00 20
-----
Screen Shot 2019-11-12 at 16 29 38
-----
Screen Shot 2019-11-12 at 22 54 50

File: src/app.jsx [hunk 1]
```diff
@@ -439,22 +440,14 @@ class Application extends React.Component {
...
         let startService = "";
         if (!this.state.systemServiceAvailable && permission.allowed) {
-            startService = <div className="alert alert-info dialog-info">
-                <div className="info-message">
-                    <span className="fa fa-exclamation-triangle" />
-                    <span>{_("System Podman service is also available")}</span>
-                </div>
-                <button onClick={this.startService}>{_("Start")}</button>
-            </div>;
+            startService = <Alert isInline variant='default'
```
Comment (marusak) [inline src/app.jsx]: I think this should not be inlined in the search panel on top, but it should be just underneath it.

File: src/app.jsx [hunk 2]
```diff
@@ -439,22 +440,14 @@ class Application extends React.Component {
...
         }
         if (!this.state.userServiceAvailable && this.state.userServiceExists) {
-            startService = <div className="alert alert-info dialog-info">
-                <div className="info-message">
-                    <span className="fa fa-exclamation-triangle" />
-                    <span>{_("User Podman service is also available")}</span>
-                </div>
-                <button onClick={this.startService}>{_("Start")}</button>
-            </div>;
+            startService = <Alert isInline variant='default'
```
Comment (marusak) [inline src/app.jsx]: as well

**Comment by KKoukiou** [comment]
@marusak how about this:
Screen Shot 2019-11-13 at 13 29 29
Or do you think they should have inline alert style?

**Comment by marusak** [comment]
> Or do you think they should have inline alert style?

I like this much more!

**Comment by garrett** [comment]
We'd need `overflow-wrap: break-word;` for the message's container. This is a PF4 bug, and we'd need to temporarily handle this in a CSS override, either globally across Cockpit (better) or for the page (not quite as good) until PF4 gets a fix upstream.

**Comment by KKoukiou** [comment]
> overflow-wrap: break-word;

It needed an 'overflow-wrap: anywhere' with the 'break-word' it didn't work.
<hr>
Screen Shot 2019-11-13 at 17 45 41
<hr>
Screen Shot 2019-11-13 at 17 45 05

**Pull request merged**